Makefile

```diff
@@ -63,6 +63,7 @@ endif
 
 # https://www.unrealengine.com/en-US/marketplace-guidelines#263b
 .PHONY: package
+
```

Comment:
This change seems unnecessary and doesn't follow the convention used elsewhere in this file

```suggestion
```